src/Microsoft.AspNet.Razor/Generator/GeneratedTagHelperContext.cs

```diff
@@ -52,21 +58,53 @@ public GeneratedTagHelperContext()
         public string ScopeManagerEndMethodName { get; set; }
 
         /// <summary>
+        /// The name of the property used to determine if a tag helper output's content was set.
+        /// </summary>
+        public string OutputContentSetPropertyName { get; set; }
```

Comment:
Unrelated suggestion: maybe using property initializers might make it easier to lookup the default names of these properties?

` public string OutputContentSetPropertyName { get; set; } = "ContentSet";`

Don't do it in this PR since it's just added noise, just a consideration.